modules/ducktests/tests/ignitetest/services/ignite.py

```diff
@@ -38,10 +38,47 @@ def __init__(self, context, config, num_nodes, jvm_opts=None, startup_timeout_se
         super().__init__(context, config, num_nodes, startup_timeout_sec, shutdown_timeout_sec, modules=modules,
                          jvm_opts=jvm_opts)
 
+    @property
+    def database_dir(self):
```

Comment:
should this be defined at IgnitePathAware?

Reply:
fixed